Update Feature-Policy to Permissions-Policy (#22439)
mdn/content: files/en-us/web/privacy/index.md

      <td><a href="/en-US/docs/Web/API/Permissions_API">Permissions API</a></td>
      <td>
        Provides a way to determine the status of permissions for the current
        browser context.
      </td>
    </tr>
    <tr>
      <td><a href="/en-US/docs/Web/HTTP/Permissions_Policy">Permissions Policy</a></td>

>       <td>
>         Lets web developers selectively enable, disable, and modify the behavior
>         of certain features and APIs both for a document and for subdocuments
>         loaded in {{HTMLElement("iframe")}}s via the {{HTTPHeader("Permissions-Policy")}} HTTP header. The {{htmlattrxref("allow", "iframe")}} attribute can be used to set Permissions Policies on individual {{HTMLElement("iframe")}}s.
>       </td>

Reviewer: I don't think you can "modify", and as above, I am not certain web developers sets the right tone. But this is my opinion.

```suggestion
      <td>
        Lets web servers selectively enable or disable features and APIs, both for a document and for subdocuments
        loaded in {{HTMLElement("iframe")}}s via the {{HTTPHeader("Permissions-Policy")}} HTTP header.
        The {{htmlattrxref("allow", "iframe")}} attribute can be used to set Permissions Policies on individual {{HTMLElement("iframe")}}s.
      </td>
```

  </tbody>
</table>

> Not all of these are generally directly used within code; notably, the [Permissions API](/en-US/docs/Web/API/Permissions_API), [Permissions Policy](/en-US/docs/Web/HTTP/Permissions_Policy), and the {{htmlattrxref("allow", "iframe")}} attribute on {{HTMLElement("iframe")}} elements are primary tools directly used by code to help secure a site and its content.

Author: Bit of a confusing and pointless paragraph. I've just deleted it.
